roles/apim-km/templates/carbon-home/repository/conf/datasources/master-datasources.xml.j2

```diff
@@ -0,0 +1,134 @@
+<datasources-configuration xmlns:svns="http://org.wso2.securevault/configuration">
+
+    <providers>
+        <provider>org.wso2.carbon.ndatasource.rdbms.RDBMSDataSourceReader</provider>
+    </providers>
+
+    <datasources>
+
+        <datasource>
+            <name>WSO2_CARBON_DB</name>
+            <description>The datasource used for registry and user manager</description>
+            <jndiConfig>
+                <name>jdbc/WSO2CarbonDB</name>
+            </jndiConfig>
+            <definition type="RDBMS">
+                <configuration>
+                    <url>jdbc:h2:repository/database/WSO2CARBON_DB;DB_CLOSE_ON_EXIT=FALSE</url>
+                    <username>wso2carbon</username>
+                    <password>wso2carbon</password>
+                    <driverClassName>org.h2.Driver</driverClassName>
+                    <maxActive>50</maxActive>
+                    <maxWait>60000</maxWait>
+                    <testOnBorrow>true</testOnBorrow>
+                    <validationQuery>SELECT 1</validationQuery>
+                    <validationInterval>30000</validationInterval>
+                    <defaultAutoCommit>true</defaultAutoCommit>
+                </configuration>
+            </definition>
+        </datasource>
+
+        <datasource>
+            <name>WSO2AM_DB</name>
+            <description>The datasource used for API Manager database</description>
+            <jndiConfig>
+                <name>jdbc/WSO2AM_DB</name>
+            </jndiConfig>
+            <definition type="RDBMS">
+                <configuration>
+                    <url>{{ wso2am_db_url }}</url>
+                    <username>{{ wso2am_db_username }}</username>
+                    <password>{{ wso2am_db_password }}</password>
+                    <defaultAutoCommit>true</defaultAutoCommit>
+                    <driverClassName>{{ wso2am_db_driver_class_name }}</driverClassName>
+                    <maxActive>50</maxActive>
+                    <maxWait>60000</maxWait>
+                    <testOnBorrow>true</testOnBorrow>
+                    <validationQuery>SELECT 1</validationQuery>
+                    <validationInterval>30000</validationInterval>
+                </configuration>
+            </definition>
+        </datasource>
+
+         <datasource>
+            <name>WSO2AM_STATS_DB</name>
+            <description>The datasource used for getting statistics to API Manager</description>
+            <jndiConfig>
+                <name>jdbc/WSO2AM_STATS_DB</name>
+            </jndiConfig>
+            <definition type="RDBMS">
+                <configuration>
+                    <url>{{ wso2am_stats_db_url }}</url>
+                    <username>{{ wso2am_stats_db_password }}</username>
+                    <password>{{ wso2am_stats_db_password }}</password>
+                    <defaultAutoCommit>true</defaultAutoCommit>
+                    <driverClassName>{{ wso2am_stats_db_driver_class_name }}</driverClassName>
```

Comment:
Isn't it enough to say `wso2am_stats_db_driver` ?

Reply:
The name of the datasource is ** <name>WSO2AM_STATS_DB</name>**. I have used the same so that the user will not get confused due to multiple names.

Reply:
I think I confused you. :) 
What I was telling is rather than using the suffix `driver_class_name`, we can just use `driver` here.